iiify/resolver.py

```diff
@@ -387,6 +387,25 @@ def create_manifest3(identifier, domain=None, page=None):
                     #                            id=f"https://iiif.archivelab.org/iiif/{identifier}${pageCount}/canvas",
                     #                            label=f"{page['leafNum']}")
                     pageCount += 1
+    
+
+            # Setting logic for paging behavior and starting canvases
+            # Start with paged (default) or individual behaviors
+            try:
```

Comment:
Maybe add comment saying that it won't be set if the book reader brOptions has something other than mode/1up